### ISSUE TITLE
[Performance] Score created in 4.1 opens in 4.0.2 two times faster than in 4.1.1/4.1.0

### ISSUE THREAD

**Issue by DmitryArefiev** [musescore/MuseScore#18784]
### Issue type

Choose option...

### Bug description

The score is taken from https://musescore.org/en/comment/1199892

### Steps to reproduce

1. Open [Shostakovich Symphony No. 4 Solo Piano.zip](https://github.com/musescore/MuseScore/files/12147016/Shostakovich.Symphony.No.4.Solo.Piano.zip) in **4.1.1** and measure the time - it takes **~20-25sec**
2. Open [Shostakovich Symphony No. 4 Solo Piano.zip](https://github.com/musescore/MuseScore/files/12147016/Shostakovich.Symphony.No.4.Solo.Piano.zip) in **4.0.2** - after clicking Open anyway in the warning dialog. it takes  **~10sec**

### MuseScore Version

4.1.1

### Regression

I don't know

### Operating system

Windows10

**Comment by DmitryArefiev** [comment]
@mike-spa Not sure is it valid test or not.. but I think it's worth to investigate it

**Comment by bkunda** [comment]
FWIW, I just tried opening this score in today's nightly and it took ca.18.5sec.

**Comment by DmitryArefiev** [comment]
@bkunda I've retested on my current laptop (Lenovo IdeaPad Pro 5 16APH8, AMD Ryzen 7 7840HS)

* Opening [Shostakovich Symphony No. 4 Solo Piano.zip](https://github.com/musescore/MuseScore/files/12147016/Shostakovich.Symphony.No.4.Solo.Piano.zip) in **4.0.2** takes  **~3sec**
* Opening [Shostakovich Symphony No. 4 Solo Piano.zip](https://github.com/musescore/MuseScore/files/12147016/Shostakovich.Symphony.No.4.Solo.Piano.zip) in **4.1.1** takes  **~22sec**
* Opening [Shostakovich Symphony No. 4 Solo Piano.zip](https://github.com/musescore/MuseScore/files/12147016/Shostakovich.Symphony.No.4.Solo.Piano.zip) in **4.4** latest master takes  **~22sec**

As you see, there are no changes in 4.4 in performance (the same result as in 4.1.1). But I think we can move to the next release

**Comment by DmitryArefiev** [comment]
@mike-spa Did you try that score on your end? Can we do something on performance side (or it's expected)?

**Comment by mike-spa** [comment]
3 seconds versus 22 seconds?? That is extremely weird. Do you get the same 10x increase in _all_ scores, or just this one?

**Comment by DmitryArefiev** [comment]
Nope. Just that score. Actually I just don't know what elements from that score take so much time
